proto/v1/org_policy_service.proto

```diff
@@ -210,6 +211,8 @@ message Policy {
     SQLReviewPolicy sql_review_policy = 11;
     SlowQueryPolicy slow_query_policy = 12;
     DisableCopyDataPolicy disable_copy_data_policy = 16;
+    MaskingRulePolicy masking_rule_policy = 17;
+    MaskingExceptionPolicy masking_exception_policy = 18;
```

Comment:
We finally decide to use two policies for rule and exception?

Reply:
Yeah.